installers/vagrant-dev/radiasoft-download.sh

```diff
@@ -40,6 +40,17 @@ vagrant_dev_box_add() {
     fi
 }
 
+vagrant_dev_disable_security() {
+    vagrant ssh <<'EOF'
+sudo bash <<'EOF_BASH'
+systemctl stop firewalld || true
+systemctl disable firewalld || true
+perl -pi -e 's{(?<=^SELINUX=).*}{disabled}' /etc/selinux/config || true
```

Comment:
I don't think you want the `|| true` for the edit. If /etc/selinux/config isn't there, perl will crash. Otherwise, perl will exit 0.

Reply:
a3494c1